workflow/cron/operator.go

```diff
@@ -303,3 +297,13 @@ func (woc *cronWfOperationCtx) deleteOldestWorkflows(jobList []v1alpha1.Workflow
 	}
 	return nil
 }
+
+func (woc *cronWfOperationCtx) reportCronWorkflowError(errString string) {
+	log.Errorf(errString)
```

Comment:
woc.log

Reply:
You should only log errors if there is something that needs to be fixed - potentially a bug in the controller. I think this is just a problem with the workflow?

Reply:
Good point, I see the distinction. Will do